responses/nl/HassBroadcast.yaml

```diff
@@ -0,0 +1,5 @@
+language: nl
+responses:
+  intents:
+    HassBroadcast:
+      default: "Klaar"
```

Comment:
Wouldn't "Gedaan" be better in this case?

Reply:
Gedaan